Entities/AHuman.cpp

```diff
@@ -3449,6 +3449,8 @@ void AHuman::Update()
 
 	if (m_Controller.IsState(WEAPON_DROP) && m_Status != INACTIVE) {
 		bool anyDropped = false;
+        bool currentlyFGArm = 1;
```

Comment:
Don't use `1` and `0` for boolean values in cpp, just use `true` and `false` instead.